### PR TITLE
Astro: Added moon age in percent and degree

### PR DESCRIPTION
**Pull request by gerrieg**
Signed-off-by: Gerhard Riegler <gerhard.riegler@gmail.com>

### CODE REVIEW

**Comment by sjka** [comment]
Thanks!

**Comment by LMS0815** [comment]
Thanks!

File: extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -332,11 +334,25 @@
 		<state readOnly="true" pattern="%d d" />
 	</channel-type>
 
+	<channel-type id="ageDegree">
+		<item-type>Number</item-type>
+		<label>Moon age in degree</label>
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml]: Note that labels should be as short as possible (best 1-2 words only).
As the unit is shown in the state anyhow, couldn't this simply be "Moon age"?

File: extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml [hunk 2]
```diff
@@ -332,11 +334,25 @@
...
+		<description>The age of the moon in degree</description>
+		<state readOnly="true" pattern="%.2f °" />
+	</channel-type>
+
+	<channel-type id="agePercent">
+		<item-type>Number</item-type>
+		<label>Moon age in percent</label>
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml]: Note that labels should be as short as possible (best 1-2 words only).
As the unit is shown in the state anyhow, couldn't this simply be "Moon age"?

**Comment by gerrieg** [comment]
OK, see https://github.com/eclipse/smarthome/pull/4835

**Comment by LMS0815** [comment]
please review / merge

**Comment by maggu2810** [comment]
> please review / merge

@LMS0815 What do you mean?

```text
sjka  merged 2 commits into eclipse:master from gerrieg:astro 14 days ago
```

**Comment by LMS0815** [comment]
OK thx. The binding is always skipped during [daily bundle](https://openhab.ci.cloudbees.com/job/openHAB2-Bundles/lastBuild/) build.
I assumed this had something to do with a non-approval.

**Pull request merged**